### PR TITLE
adding documentation for gist 12

### PR DESCRIPTION
**Pull request by pwin**

### CODE REVIEW

**Review by sa-bpelakh — COMMENTED** [review]
This can't be reviewed like a normal PR, given that's a live HTML document (and a epub, for which I need to find a reader). but I have a couple of questions:
1. Why are you removing the gist 11-0 docs, as opposed to just archiving it? Sometimes people might need to reference old documentation, if they have not yet upgraded.
2. Why are we putting d3 and webowl under our source control, as opposed to loading them from their normal locations? I am not even sure if we are allowed to redistribute them in this fashion.

**Comment by pwin** [comment]
so should we turn this into a website ?

**Comment by Jamie-SA** [comment]
https://gist-doc.semanticarts.com/

Maybe it should get added to the README.

**Comment by rjyounes** [comment]
> Why are you removing the gist 11-0 docs, as opposed to just archiving it? Sometimes people might need to reference old documentation, if they have not yet upgraded.

I agree 100%. I had all along envisioned that we would keep all prior documentation in versioned folders.

File: README.md [hunk 1]
```diff
@@ -3,21 +3,20 @@ gist-doc
 
 This repository contains documentation of the [Semantic Arts gist minimalist upper ontology](https://github.com/semanticarts/gist/tree/master) in three formats: narrative, graphical, and Widoco-style.
 
-Current version of gist: 11.0
+Current version of gist: 12.0.0
```
Comment (rjyounes) [inline README.md:6]: Are we going to refresh this only at major releases? And not have to update this line every time we publish a new version of gist? Then we should say 

Current major version of gist: 12

Or 

Current version of gist: 12.x.x ("Current version" not quite right here, I don't know how to refer to this).

File: README.md [hunk 2]
```diff
@@ -3,21 +3,20 @@ gist-doc
...
 Contents
 -----
 
-* **gist Constellations (eBook)**
-  * The organization of the eBook is based around the idea of 'constellations' of classes and predicates that deal with related concepts.  
-  * epub and mobi formats
-* **gist Constellations (visualizations)**
-  * A collection of images built using the [Turtle Editor Viewer](http://semantechs.co.uk/turtle-editor-viewer/).  Each image is made up of a single 'constellation' of classes as described above and shows the classes and their relationships, the literal annotations, and any anonymous classes used in class definitions.
+* **gist eBook**
+  * The organization of the eBook is based around the idea of themes of classes and predicates that deal with related concepts.
```
Comment (rjyounes) [inline README.md:12]:
```suggestion
  * The eBook is organized around conceptual themes, such as addresses, magnitudes, and so on.
```

File: README.md [hunk 3]
```diff
@@ -3,21 +3,20 @@ gist-doc
...
 -----
 
-* **gist Constellations (eBook)**
-  * The organization of the eBook is based around the idea of 'constellations' of classes and predicates that deal with related concepts.  
-  * epub and mobi formats
-* **gist Constellations (visualizations)**
-  * A collection of images built using the [Turtle Editor Viewer](http://semantechs.co.uk/turtle-editor-viewer/).  Each image is made up of a single 'constellation' of classes as described above and shows the classes and their relationships, the literal annotations, and any anonymous classes used in class definitions.
+* **gist eBook**
+  * The organization of the eBook is based around the idea of themes of classes and predicates that deal with related concepts.  
+  * epub format distribution of the "Zest of gist" eBook will work with any modern eBook reader including Calibre and Kindle reader
```
Comment (rjyounes) [inline README.md:13]:
```suggestion
  * The epub distribution will work with any modern eBook reader, including Calibre and Kindle reader.
```

File: README.md [hunk 4]
```diff
@@ -3,21 +3,20 @@ gist-doc
...
 
 Contents
 -----
 
-* **gist Constellations (eBook)**
-  * The organization of the eBook is based around the idea of 'constellations' of classes and predicates that deal with related concepts.  
-  * epub and mobi formats
-* **gist Constellations (visualizations)**
-  * A collection of images built using the [Turtle Editor Viewer](http://semantechs.co.uk/turtle-editor-viewer/).  Each image is made up of a single 'constellation' of classes as described above and shows the classes and their relationships, the literal annotations, and any anonymous classes used in class definitions.
+* **gist eBook**
```
Comment (rjyounes) [inline README.md:11]:
```suggestion
* **gist eBook: _The Zest of Gist_**
```
Not sure if this formatting works. I'm going for the title in italics inside the bold.

File: README.md (hunk 1 above)
Comment (rjyounes) [reply README.md:6]: I would also name the eBook in some way that doesn't make it out of date with patch or minor versions.

File: gist-12.0/widoco-documentation/readme.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+About Widoco output
```
Comment (rjyounes) [inline gist-12.0/widoco-documentation/readme.md:1]:
```suggestion
About Widoco Output
```

File: gist-12.0/widoco-documentation/readme.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+===================
```
Comment (rjyounes) [inline gist-12.0/widoco-documentation/readme.md:2]: Why do we need documentation of widoco itself? We are not expecting anyone to create the widoco documentation. When they view the widoco documentation they will see immediately what it is and how it is structured.  If we really want, we can just link to existing widoco documentation, say here: https://github.com/dgarijo/Widoco. I recommend removing the entire section.

**Review by rjyounes — CHANGES_REQUESTED** [review]

**Comment by rjyounes** [comment]
> https://gist-doc.semanticarts.com/

This is a good initial version. I would like to see two things in the future:
* Standardize on gist-doc or gist-docs (singular vs plural). We are using each in various places - repo name, link, HTML title, etc. I don't have a preference but I like consistency.
* At some point add some CSS to the web page.

**Comment by rjyounes** [comment]
Closing - superseded by PR #10.

**Pull request closed without merging**